Include signal.h to fix:
libs/std/process.c:461:2: error: implicit declaration of function 'kill' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
https://github.com/HaxeFoundation/neko/commit/2435d03cb33bf00acdb8831986c6e3b33fe78401
--- libs/std/process.c.orig
+++ libs/std/process.c
@@ -27,6 +27,7 @@
 #	include <sys/types.h>
 #	include <unistd.h>
 #	include <errno.h>
+#	include <signal.h>
 #	if !defined(NEKO_MAC)
 #		if defined(NEKO_BSD)
 #			include <sys/wait.h>
